I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scripts/script.rpy", line 40, in script
    $testVar = GetAttitudeBinary(EVatt)
  File "game/scripts/script.rpy", line 40, in <module>
    $testVar = GetAttitudeBinary(EVatt)
  File "game/scripts/script.rpy", line 70, in GetAttitudeBinary
    if GetAttitude(chatatt) < 1:
NameError: name 'chatatt' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scripts/script.rpy", line 40, in script
    $testVar = GetAttitudeBinary(EVatt)
  File "C:\Users\qwawe\Downloads\renpy-8.0.3-sdk\renpy-8.0.3-sdk\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\qwawe\Downloads\renpy-8.0.3-sdk\renpy-8.0.3-sdk\renpy\python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/scripts/script.rpy", line 40, in <module>
    $testVar = GetAttitudeBinary(EVatt)
  File "game/scripts/script.rpy", line 70, in GetAttitudeBinary
    if GetAttitude(chatatt) < 1:
NameError: name 'chatatt' is not defined

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
Novel_URFU_22 1.0
Sat Nov 26 00:57:25 2022
